.kitchen.yml

```diff
@@ -131,3 +131,10 @@ suites:
       systems:
         - name: workload_metadata_config
           backend: local
+  - name: "simple_regional_skip_local_exec"
```

Comment:
This doesn't need to be a separate suite, just add it to one of the existing ones.

Reply:
Moved to `simple_regional`